### PR TITLE
small fixes for actions UI

### PR DESCRIPTION
**Pull request by roperzh**
OK to merge.

Short review.

Summary of changes:

Fixes https://app.asana.com/0/680508544654148/700874348545631

- Don't show main as having actions if it only has timeline actions
- Change the shade of blue of the bolt icon

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [x] Added measurement instrumentation (Mixpanel, etc.)
- [x] Wrote an automated test covering new functionality

### CODE REVIEW

**Comment by roperzh** [comment]
screen shot 2018-06-12 at 13 55 02

I am seeing this ☝️ but the branch is rebased against `rc-3.4.5.2` 🤷‍♂️

**Comment by stristr** [comment]
I think the problem is `rc-3.4.5.2`, which is apparently not rebased against `master`…will check later.

**Comment by roperzh** [comment]
Update: pushed a commit  to also fix https://app.asana.com/0/680508544654148/697640433446165

**Comment by stristr** [comment]
LGTM (so far)! Except for bad Git history, but let's just squash-merge this and really fix in `rc-3.4.5.2`.

**Comment by roperzh** [comment]
Update II: pushed another commit to fix https://app.asana.com/0/680508544654148/704795620573338

File: packages/haiku-timeline/src/components/Timeline.js [hunk 1]
```diff
@@ -1207,8 +1206,8 @@ class Timeline extends React.Component {
                                 getActiveComponent={() => {
                                   return this.getActiveComponent()
                                 }}
-                                showEventHandlersEditor={() => {
-                                  this.showEventHandlersEditor()
+                                showEventHandlersEditor={(...args) => {
```
Comment (stristr) [inline packages/haiku-timeline/src/components/Timeline.js:1209]: I think the fix we want here is simply:

```showEventHandlersEditor={this.showEventHandlersEditor}```

When we typescriptify, I'm adding the `jsx-no-lambda` rule, so we'll actually be forbidding this. In TypeScript, the preferred way will be:

```
private boundShowEventHandlers = (...args) => this.showEventHandlers(...args);
```

In JS, the `.bind(...)` pattern is fine.

Sorry for mixed messages here! We still want to prefer lambda functions, but _not_ create them inside `render()` method for performance reasons.

Comment (stristr) [reply packages/haiku-timeline/src/components/Timeline.js:1209]: (Docs if interested:

> Creating new anonymous functions (with either the `function` syntax or ES2015 arrow syntax) inside the `render` call stack works against _pure component rendering_. When doing an equality check between two lambdas, React will always consider them unequal values and force the component to re-render more often than necessary.

)

**Comment by stristr** [comment]
Merging.

File: packages/haiku-timeline/src/components/Timeline.js (hunk 1 above)
Comment (roperzh) [reply packages/haiku-timeline/src/components/Timeline.js:1209]: kk sounds good! 👍

**Pull request merged**